### ISSUE TITLE
[Feature] 3D Support in Stash

### ISSUE THREAD

**Issue by Noob34567** [stashapp/stash#2384]
I am new to Stash and I really enjoy using the app. Thanks for the effort from all developers.

There are other solutions, such as XBVR, to watch VR movies. There are many reasons why I like Stash more then other apps that I tested, but the lack of VR movies make it difficult for me to switch to stash.

The DeoVR Plugin can just export one list, I would like to filter in 3D Mode, i.e. using the "normal" scene view. 

Therefore, would it be possible to enhance the video player in a way that it can play 3D videos?

I am using the GearVR, that might make it more difficult, because GearVR is not supported anymore. 

I tried to open stash with DeoVR, also the "normal" stash app, without the DeoVR plugin. I also tried the Browser, but due to lack of updates in GearVR it might be difficult to use.

Thanks in advance....

**Comment by philpw99** [comment]
Welcome my fellow DeoVR user ! I faced the same problems like you when I started using Stash.
6 months later, I had 2 solutions for the 3D VR playback with Stash.
1. Use my [Stash_Helper](https://github.com/philpw99/Stash_Helper/releases) and set the default media player as "DeoVR".
2. Use my [special built Stash](https://github.com/philpw99/stash/releases/tag/v0.12.2) to play videos directly from the browser in DeoVR. ( It's a variation of v.0.12. Not an official release. So you cannot update it. )
Either solution will help you play 3D videos with Stash.

I never used GearVR so I don't know if the above will help.
